apps/admin/frontend/src/components/import_cvrfiles_modal.tsx

```diff
@@ -266,25 +266,31 @@ export function ImportCvrFilesModal({ onClose }: Props): JSX.Element | null {
   }
 
   if (currentState.state === 'success') {
+    const { alreadyPresent, newlyAdded } = currentState.result;
+    const total = alreadyPresent + newlyAdded;
+    const content = (() => {
+      if (alreadyPresent > 0) {
+        if (total === 1) {
+          return <P>The 1 CVR in the selected export was previously loaded.</P>;
+        }
+        return (
+          <P>
+            Of the {format.count(total)} total CVRs in the selected export,{' '}
+            {format.count(alreadyPresent)}{' '}
+            {alreadyPresent === 1 ? 'was' : 'were'} previously loaded.
+          </P>
+        );
+      }
+      return <P>The CVRs in the selected export were successfully loaded.</P>;
```

Comment:
Aside from better handling singular vs. plural cases, I've added content for the basic, no-duplicates success case. We used to have no content for the success case, just a title. Which made the modal look a bit funny IMO. I've added content for consistency with the other cases.

_Before / After_

<img width="300" alt="before" src="https://github.com/votingworks/vxsuite/assets/12616928/3ebb09c3-da30-430a-9a1a-c8b356bb8865"> <img width="300" alt="after" src="https://github.com/votingworks/vxsuite/assets/12616928/7c374bba-274b-43f4-b9d3-e98a7c8466a5">